### PR TITLE
doc: clarify & fix documentation of :extra-pages config option

### LINKED ISSUE #2070: config-default.edn contains syntaxt error
Hi,

Instead of `file` it should be `filename` I believe. `file` results in application startup errors.
https://github.com/CSCfi/rems/blob/master/resources/config-defaults.edn#L82

I'd also recommend not using highly edn specific syntax in this example file (or document it better) as most admins will likely not know what `#_ ` stands for and are confused when the settings are not applied.

### PR DESCRIPTION
**Pull request by opqdonut**
fixes #2070

### CODE REVIEW

File: resources/config-defaults.edn [hunk 1]
```diff
@@ -75,18 +75,18 @@
...
-                                 :en {:title "Hello"}}}
-
-               ;; example of a local markdown file
-               #_{:id "about"
-                  :translations {:fi {:title "Info"
-                                      :file "about-fi.md"}
-                                 :en {:title "About"
-                                      :file "about-en.md"}}}]
+ ;;
+ ;; Example: defines an external page and a markdown page:
```
Comment (Macroz) [inline resources/config-defaults.edn]: How about something like this:
`;; Example: define an external static HTML page that will be linked to`
And the other with
`;; Example: a page generated from local Markdown files`

Comment (opqdonut) [reply resources/config-defaults.edn]: I wanted to demonstrate how you can have multiple definitions in an array. I'll try to make it clearer....

Comment (Macroz) [reply resources/config-defaults.edn]: Yeah just thought that it would be clearer if you split them and described what they actually are

Comment (opqdonut) [reply resources/config-defaults.edn]: how about now?

**Pull request merged**